cradle-core/build.gradle

```diff
@@ -1,7 +1,6 @@
 dependencies {
-	compile "org.slf4j:slf4j-api:${slf4j_version}"
-	compile "org.slf4j:slf4j-log4j12:${slf4j_version}"
-	compile "org.slf4j:slf4j-simple:${slf4j_version}"
+	implementation "org.slf4j:slf4j-api:${slf4j_version}"
+	testImplementation "org.slf4j:slf4j-simple:${slf4j_version}"
 	compile "com.fasterxml.jackson.core:jackson-databind:${jackson_version}"

```

Comment:
I think while we are here it is better to replace `compile` and `testCompile` configurations to `implementation` and `testImplementation` respectively. The old one are deprecated in newer Gradle versions